tests/test_api.py

```diff
@@ -0,0 +1,27 @@
+"""Test api.py functionality."""
+import pytest
+from os.path import isfile
+
+import pos
+from pos.types import Dataset
+
+MODEL_LOCATION = "tagger.pt"
+DICT_LOCATION = "dictionaries.pickle"
+
```

Comment:
Shouldn't the model and dictionary location be flexible like in https://github.com/cadia-lvl/punctuation-prediction/pull/5 Otherwise everyone running this test will always need to specify their model and dictionary locations

Reply:
I've added some options to the pytest command so that users can specify the location of these files when running the tests.